app/controllers/outgoings_controller.rb

```diff
@@ -0,0 +1,38 @@
+class OutgoingsController < ApplicationController
+  # api! works but the links in resulting docs are broken. See https://github.com/Apipie/apipie-rails/issues/559
+  api :POST, 'assessments/:assessment_id/outgoings', 'Create outgoings'
+  formats ['json']
+  param :assessment_id, :uuid, required: true
+  param :outgoings, Array, desc: 'Collection of outgoings', required: true do
+    param :type_of_outgoing, Outgoing.type_of_outgoings.values, required: true, desc: 'The type of product or service the outgoing paid for'
+    param :payment_date, Date, date_option: :today_or_older, required: true, desc: 'The date the outgoing was paid'
+    param :amount, :currency, required: true, desc: 'The monetary amount paid for the outgoing'
+  end
+
+  returns code: :ok, desc: 'Successful response' do
+    property :objects, array_of: Outgoing
+    property :success, ['true'], desc: 'Success flag shows true'
+  end
+  returns code: :unprocessable_entity do
+    property :error, String, desc: 'Description of why object invalid'
```

Comment:
I think errors should be an array like we decided in our slack conversation with the other devs